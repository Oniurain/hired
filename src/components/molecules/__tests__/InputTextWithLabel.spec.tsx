import * as React from 'react'
import { mount } from 'enzyme'

import { InputTextWithLabel } from '../'

describe('InputTextWithLabel', () => {
  it('should render', () => {
    const subject = mount(
      <InputTextWithLabel
        id="description"
        label="Description"
        placeholder="Enter a description for the current project"
        changeHandler={() => { }}
      />
    )

    expect(subject).toMatchSnapshot()
  })
})
