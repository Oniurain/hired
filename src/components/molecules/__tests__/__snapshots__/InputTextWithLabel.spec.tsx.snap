// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputTextWithLabel should render 1`] = `
<InputTextWithLabel
  changeHandler={[Function]}
  id="description"
  label="Description"
  placeholder="Enter a description for the current project"
>
  <Styled(BaseRow)>
    <BaseRow
      className="dHOzQD"
    >
      <div
        className="dHOzQD"
      >
        <styled.label
          htmlFor="description"
        >
          <label
            className="fPhlMm"
            htmlFor="description"
          >
            Description
          </label>
        </styled.label>
        <styled.input
          id="description"
          onChange={[Function]}
          placeholder="Enter a description for the current project"
          type="text"
        >
          <input
            className="iERWm"
            id="description"
            onChange={[Function]}
            placeholder="Enter a description for the current project"
            type="text"
          />
        </styled.input>
      </div>
    </BaseRow>
  </Styled(BaseRow)>
</InputTextWithLabel>
`;
